lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_action.rb

```diff
@@ -27,7 +27,8 @@ def self.run(params)
           target: params[:target],
           description: release_notes,
           assets: assets,
-          prerelease: prerelease
+          prerelease: prerelease,
+          githubtoken: params[:githubtoken]
```

Comment:
💄 Nit: Ruby naming conventions use snake_case for parameters and variable names.
```suggestion
          github_token: params[:github_token]
```

Reply:
Thanks, I'll address that!